### PR TITLE
Add textarea

### PR DESCRIPTION
**Pull request by raczyk**

### CODE REVIEW

File: src/components/textarea/Textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  font-size: ${(props: TextareaProps) =>
+    props.small ? theme.fontSizes.sm : theme.fontSizes.md};
+  line-height: ${(props: TextareaProps) =>
+    props.small ? "1rem" : theme.lineHeight};
+  padding: 0;
+  margin: 0;
+  transition: ${theme.transition};
+  color: ${theme.color.text.text01};
+  resize: none;
+  &::placeholder {
```
Comment (nowyDEV) [inline src/components/textarea/Textarea.tsx:64]: add empty line above

File: src/components/textarea/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    background: transparent;
+    border: 0;
+    cursor: not-allowed;
+  }
+`;
+
+type Props = TextareaGroupProps & TextareaProps;
+
+export const Textarea: React.FC<Props> = React.forwardRef(
+  ({ status, small, ...props }, ref) => {
```
Comment (nowyDEV) [inline src/components/textarea/Textarea.tsx:85]: small can also be spreaded in the rest props

**Review by nowyDEV — APPROVED** [review]

**Review by Horay — DISMISSED** [review]

File: src/components/textarea/Textarea.tsx (hunk 2 above)
Comment (raczyk) [reply src/components/textarea/Textarea.tsx]: done

File: src/components/textarea/Textarea.tsx (hunk 1 above)
Comment (raczyk) [reply src/components/textarea/Textarea.tsx:65]: done

**Review by nowyDEV — APPROVED** [review]

**Review by Horay — APPROVED** [review]

**Pull request merged**